vega/static/vega-lite.js

```diff
@@ -2,6 +2,10 @@ var spec = {spec};
 var selector = "{selector}";
 var type = "{type}";
 
+console.log(spec);
+console.log(selector);
+console.log(type);
+
```

Comment:
Typically all console.log statements should be removed. I think they are only relevant for debugging.